tests/services/mapping_api_service.py

```
@@ -0,0 +1,91 @@
+                "row_number": idx,
+                "nomenclature": case['Номенклатура поставщика']
+            } for idx, case in enumerate(test_cases)
+        ]
+    }
+
+    response = requests.post(API_URL, json=payload, headers=headers)
+
+    if response.status_code == 200:
+        return response.json().get("job_id")
+    else:
+        logger.info(f"Failed to start mapping")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)